tests/unittests/server/routes/test_sample.py

```
@@ -938,6 +939,229 @@ async def test_patch_add_mask_persists_as_numpy(
+        sample.reload()
+        det = sample.ground_truth.detections[0]
+        assert det.mask is None
+        assert det.mask_path == mask_file
+
+        # Verify disk: file was overwritten with edited mask
+        after = _read_mask(mask_file)
+        assert after.max() > 0, "file should now have content"
+
+    @pytest.mark.asyncio
+    @pytest.mark.parametrize(
+        "field, det_id_fn, status, description",
```

Ruff (0.15.11)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)